src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -96,6 +105,7 @@ public DefaultCache(
                 counters.supplyCounter.apply("http-cache.cached.response.acquires"),
                 counters.supplyCounter.apply("http-cache.cached.response.releases"));
         this.cachedEntries = new Int2ObjectHashMap<>();
+        this.links = new Int2ObjectHashMap<>();
```

Comment:
Rename `cachedEntries` to `cachedEntriesByRequestHash`.
Rename `links` to `cachedEntriesByRequestHashWithoutQuery`.

Reply:
Done.